src/software/jetson_nano/thunderloop.cpp

```diff
@@ -271,21 +286,26 @@ Thunderloop::~Thunderloop() {}
             redis_client_->asyncCommit();
         }
 
-        auto loop_duration =
-            iteration_time.tv_sec * static_cast<int>(NANOSECONDS_PER_SECOND) +
-            iteration_time.tv_nsec;
-        thunderloop_status_.set_iteration_time_ns(loop_duration);
+        auto loop_duration = getMilliseconds(iteration_time);
```

Comment:
we might want to keep `loop_duration` in ns for logic on line 298 to also makes sense and have the correct units

Reply:
oh yeah this was supposed to be changed since the function now returns milliseconds